<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolvingEquationsTab</class>
 <widget class="QWidget" name="SolvingEquationsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>539</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="resultsLabel">
     <property name="text">
      <string>Results:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="equationsLabel">
     <property name="text">
      <string>Number of equations:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="variableNamesWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="variableNamesList">
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="equationsWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="equationsTable">
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTableView" name="resultsTable">
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SolvingEquationsTab</receiver>
   <slot>changeSpinBoxValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SolvingEquationsTab</receiver>
   <slot>calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeSpinBoxValue(int)</slot>
  <slot>calculate()</slot>
 </slots>
</ui>
